Максонов Артем Сергеевич

BloodMagicLord opened the pull request.

Commit:
task-1 done

marikakosareva commented on about-cats/src/index.html:
```diff
@@ -5,7 +5,161 @@
+        <a>Напишите нам:</a>
+        <ul>
+          <li>
+            <a href="mailto:cats@babuli.ru" class="email">cats@babuli.ru</a>
```
ссылки должны быть без подчеркивания

marikakosareva commented on about-cats/src/index.css:
```diff
@@ -1 +1,348 @@
+  color: var(--title-color);
+}
+
+.gallery-item-description {
```
текст в карточках должен быть ориентирован по центру

marikakosareva commented on about-cats/src/index.css:
```diff
@@ -1 +1,348 @@
+.gallery-item-description {
+  margin-left: 10px;
+  margin-bottom: 10px;
+}
```
Отступы между строчками текста в карточках не соответствуют макету

marikakosareva commented on about-cats/src/index.css:
```diff
@@ -1 +1,348 @@
+  min-width: 800px;
+  margin: auto;
+  overflow: auto;
+  background: var(--background-color);
```
у карточки не должно быть белой подложки

marikakosareva commented on about-cats/src/index.html:
```diff
@@ -5,7 +5,161 @@
+        <div class="carousel-all-modals">
+          <div class="carousel-modal carousel-modal-1">
+            <div class="carousel-modal-content">
+              <label class="carousel-modal-close clickable" for="item-modal-1"></label>
```
крестик не работает, при нажатии модалка остается открытой

marikakosareva commented on about-cats/src/index.html:
```diff
@@ -5,7 +5,161 @@
-  <body>
-    <!-- TODO -->
+  <body class="body">
+    <header class="header">
```
хедер частично уходит за экран при скролле

marikakosareva reviewed (commented):
Задание выполнено. Есть мелкие правки и карточка подробного описания не закрывается при нажатии на крестик

trixartem commented:
Жду исправления комментариев

Commit:
bugs fixed

Undreline of email removed. Modal window text align is center now. Modal window background removed. Indent in card description fiex. Header bug fixed

BloodMagicLord commented:
Исправил недочёты.

trixartem commented:
✅ Принято
